### PR TITLE
Create Table

### PR DESCRIPTION
**Pull request by lukman155**
* [x]  Create a database named vet_clinic.
* [x]  Create a table named animals with the following columns:
   id: integer
   name: string
   date_of_birth: date
   escape_attempts: integer
   neutered: boolean
   weight_kg: decimal
* [x]   Query to find all animals whose name ends in "mon".
1-mon-query

 * [x]    Query to List the name of all animals born between 2016 and 2019.
2-year

 * [x]  Query to list the name of all animals that are neutered and have less than 3 escape attempts.
3-less

 * [x]  Query to list the date of birth of all animals named either "Agumon" or "Pikachu".
4-year

 * [x]  Query to list name and escape attempts of animals that weigh more than 10.5kg
5-name

 * [x]  Query to find all animals that are neutered.
6-neuter

* [x]  Query to find all animals not named Gabumon.
7-not

* [x]  Query to find all animals with a weight between 10.4kg and 17.3kg
8-within

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+CREATE TABLE animals (
+    id INT, 
+    name CHAR(24), 
+    date_of_birth DATE, 
+    escape_attempts INT, 
+    neutered BOOLEAN, 
+    weight_kg REAL)
```
Comment (sja-thedude) [inline schema.sql]: - [ ] Kindly use decimal or numeric as its valid data type for weight_kg.

File: queries.sql [hunk 2]
```diff
@@ -0,0 +1,2 @@
+
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
```
Comment (sja-thedude) [inline queries.sql:1]: - [ ] Kindly add the queries for all the questions given in the requirements.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @lukman155, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Correct Gitflow
✔️ Professional Readme

## Required 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+SELECT * FROM animals WHERE name LIKE '%mon%';
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-12-31' AND '2019-01-01';
```
Comment (mckent05) [inline queries.sql:3]: **OPTIONAL** - In this query you are to list the name of all animals born between 2016 and 2019. .ie between `2016/01/01 and 2019/31/12`. Please fix this 🙏

**Review by mckent05 — APPROVED** [review]
Hi @lukman155, 

# STATUS: APPROVED ✔️ ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well done! 🎉 
approved


## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👏 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**